docs/changelog.md

```diff
@@ -235,30 +261,41 @@ description: |
     arithmetic is needed, such as for finance
   - Added `std` module for accessing standard library definitions even when a
     variable with the same name shadows/overwrites it
+    <!-- I can't find this module in the search. It should have a docs page. -->
   - Added [`array.to-dict`], [`array.reduce`], [`array.windows`] methods
   - Added `exact` argument to [`array.zip`]
   - Added [`arguments.at`] method
   - Added [`int.from-bytes`], [`int.to-bytes`], [`float.from-bytes`], and
     [`float.to-bytes`]
-  - [`calc.round`] no longer accepts negative digits (**Minor breaking change**)
+  - `digits` parameter of [`calc.round`] no longer accepts negative integers
+    (**Minor breaking change**)
   - Conversions from [`int`] to [`float`] will now error instead of saturating
     if the float is too large (**Minor breaking change**)
+    <!-- Pretty sure docs should be updated here:
+         https://staging.typst.app/docs/reference/foundations/float/#constructor
+    -->
   - Added `float.nan` and `float.inf`, removed `calc.nan`
     (**Minor breaking change**)
+    <!-- Docs missing a simple example of printing and/or making these values. -->
   - Certain symbols are now generally callable like functions and not only
     specifically in math. Examples are accents or `floor` and `ceil`.
+    <!-- Add some link/examples.
+         https://staging.typst.app/docs/reference/math/lr#functions-ceil
+         https://staging.typst.app/docs/reference/foundations/calc#functions-ceil
+    -->
   - Improved [`repr`] of relative values, sequences, infinities, NaN,
     `{type(none)}` and `{type(auto)}`
   - Fixed crash on whole packages (rather than just files) cyclically importing
     each other
-  - Fixed behaviour of [`calc.round`] on integers when a non-zero value is
+  - Fixed behavior of [`calc.round`] on integers when a non-zero value is
     provided for `digits`
+    <!-- I wasn't able to reproduce this. -->
 
 - Styling
   - Text show rules now match across multiple text elements
   - The string `{"}` in a text show rule now matches smart quotes
   - Fixed a long-standing styling bug where the header and footer would
-    incorrectly inherit styles from a lone element on the page (e.g. a heading)
+    incorrectly inherit styles from a lone element on the page (e.g., a heading)
```

Comment:
I would prefer to not make this change now. Maybe in the future consistently across the codebase.

Reply:
Are you referring to the whole British → American part? I can do that. Does the future just mean post-0.12?

Reply:
I meant specifically `e.g.,` because the change surface is so large and it's also used a lot across the codebase. Future means simply that I don't want to deal with it right now.

Reply:
Got it. I agree that the comma change is crazy widespread. It pollutes the diff quite nicely. I will try making a list of stuff we've discussed here and then open issues/PR for them. But I don't know when I will have time for this.